### DISCUSSION TITLE
Invariant mass for dilepton and dineutrino

### DISCUSSION THREAD

**Discussion by fotonfoton123** [MadAnalysis/madanalysis5#106] [General]
Dear Authors,

I examine the processes e e > ve ve~ (here, ve is electron neutrino) and ee>ee. I want to draw plots of the events-Minv(ee) and events-Mint(veve~). What codes should I use to draw these plots?
For example ( plot M(e[1] e[2])...?)

Best Regards.

**Comment by BFuks** [comment]
Hi,

What you propose should work after indicating to the code that initial-state particles should be used (please see the manual or the reference cards).

Cheers,

Benjamin
